ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -74,6 +85,12 @@ open class CommandBar: UIView {
     public override func layoutSubviews() {
         super.layoutSubviews()
 
+        scrollView.contentInset = scrollViewContentInset()
```

Comment:
This definitely shouldn't be necessary. We shouldn't be triggering redundant layout passes during a layout pass.
The content inset setting should also only happen when necessary, not during every layout pass.

Reply:
Clarification - this applies to this line and all the layoutIfNeeded calls below.